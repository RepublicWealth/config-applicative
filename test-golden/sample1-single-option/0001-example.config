--- INI
--- ENV
--- CMDLINE
--config-example
