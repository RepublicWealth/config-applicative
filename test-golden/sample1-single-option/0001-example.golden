[XXX]

# Override using:
#   - the SAMPLE1_XXX_ABC environment variable
#   - the --XXX.abc command line option
abc = <ARG>
